src/common/ipc.ts

```diff
@@ -29,16 +29,21 @@ export function broadcastMessage(channel: string, ...args: any[]) {
 
   if (!views) return;
 
-  views.forEach(webContent => {
+  views.forEach(async webContent => {
     const type = webContent.getType();
 
     logger.silly(`[IPC]: broadcasting "${channel}" to ${type}=${webContent.id}`, { args });
     webContent.send(channel, ...args);
-    getSubFrames().then((frames) => {
-      frames.map((frameInfo) => {
+
+    try {
+      const frames = await getSubFrames();
```

Comment:
This actually changes how message ordering goes (just a bit but it might break expected behaviour).. Please revert unless there is a real reason why this is done (I tried to make similar change in #1957 but it did break some of the app functionality).

Reply:
How does this change the messaging order? I didn't make line 32 a `for-loop` so these will still happen asynchronously (though actually concurrently because JS is single-treaded)

Reply:
Based on discussions & testing it seems that this does not have side-effects. So probably fine, although I still don't see how this is related to this specific PR.

Reply:
I don't remember either. Will pull out and put into own PR